src/main/resources/transports/transports.tsv

```diff
@@ -4547,3 +4547,19 @@
 # Lletya Entrance								
 2304 3194 0	2306 3194 0	Pass Tree 8742					517>1	5
 2306 3194 0	2304 3194 0	Pass Tree 8742					517>1	5
+								
+# Mountain Guide (Varrock Museum)
+1401 3536 0	1275 3559 0	Mount Quidamortem 7600						7
+1401 3536 0	1272 3475 0	South of Quidamortem 7600						7
+1275 3559 0	1401 3536 0	The Shayzien Outpost 7600						7
+1275 3559 0	1272 3475 0	South of Quidamortem 7600						7
+1272 3475 0	1401 3536 0	The Shayzien Outpost 7600						7
+1272 3475 0	1275 3559 0	Mount Quidamortem 7600						7
+
+# Custodia Mountains Cave passage
+1259 3430 0	1271 3436 0	Pass-through Cave 57219						4
+1271 3436 0	1259 3430 0	Pass-through Cave 57219						4
+
+# Mountain Guide (Varlamore)
+1486 3232 0	1361 3309 0	Follow Mountain Guide 14529						5
+1361 3309 0	1486 3232 0	Follow Mountain Guide 14529						5
```

Comment:
I think these don't belong here, since they're NPC related not ObjectID
Maybe the best solution would be to add a new transport TSV file for NPC related transports